Jenkinsfile

```diff
@@ -0,0 +1,229 @@
+pipeline_name="vivarium"
+conda_env_name="${pipeline_name}-${BUILD_NUMBER}"
+conda_env_path="/tmp/${conda_env_name}"
+// defaults for conda and pip are a local directory /svc-simsci for improved speed.
+// In the past, we used /ihme/code/* on the NFS (which is slower)
+shared_path="/svc-simsci"
+
+
+pipeline {
+  // This agent runs as svc-simsci on node simsci-slurm-sbuild-p01.
+  // It has access to standard IHME filesystems and singularity
+  agent { label "svc-simsci" }
+
+  options {
+    // Keep 100 old builds.
+    buildDiscarder logRotator(numToKeepStr: "100")
+
+    // Wait 60 seconds before starting the build.
+    // If another commit enters the build queue in this time, the first build will be discarded.
+    quietPeriod(60)
+
+    // Fail immediately if any part of a parallel stage fails
+    parallelsAlwaysFailFast()
+  }
+
+  parameters {
+    booleanParam(
+      name: "DEPLOY_OVERRIDE",
+      defaultValue: false,
+      description: "Whether to deploy despite building a non-default branch. Builds of the default branch are always deployed."
+    )
+    booleanParam(
+      name: "IS_CRON",
+      defaultValue: true,
+      description: "Indicates a recurring build. Used to skip deployment steps."
+    )
+    string(
+      name: "SLACK_TO",
+      defaultValue: "simsci-ci-status",
```

Comment:
Note: The Jenkins pipeline should slack the test channel for all actions so we can develop but the cron pipeline in the future will post to this channel.